internal/testing/testprovider/list_resource.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package testprovider
```

Comment:
`testprovider` and `testsdk` are internal implementations based on terraform-plugin-go. They are independent of SDKv2 and Framework.